packages/manager/modules/pci/src/projects/project/failover-ips/order/agora/order.controller.js

```diff
@@ -0,0 +1,60 @@
+import JSURL from 'jsurl';
+import filter from 'lodash/filter';
+import find from 'lodash/find';
+import get from 'lodash/get';
+import isNumber from 'lodash/isNumber';
+import min from 'lodash/min';
+import { ORDER_URL } from './order.constants';
+
+export default class FailoverIpController {
+  /* @ngInject */
+  constructor(
+    $window,
+    OvhApiOrderCloudProjectIp,
+  ) {
+    this.$window = $window;
+    this.OvhApiOrderCloudProjectIp = OvhApiOrderCloudProjectIp;
+  }
+
+  $onInit() {
+    this.ip = {
+      quantity: 1,
+      instance: null,
+      region: null,
+      product: null,
+    };
+
+    const [product] = this.products;
+    const configurations = get(product, 'details.product.configurations');
+
+    this.REGIONS = get(find(configurations, { name: 'country' }), 'values');
+  }
+
+  static getMaximumQuantity(product) {
+    get(
```

Comment:
What do you think about creation multiple `const` assignments in order to keep readability and prevent nested too much Lodash methods.